src/gh_pr/core/graphql.py

```
@@ -0,0 +1,458 @@
+
+        Returns:
+            Tuple of (success, error_message)
+        """
+        # Input validation
+        if not suggestion_id:
+            return False, "Suggestion ID is required"
+
+        # Security: Validate suggestion_id format (base64 encoded GitHub ID)
+        # Base64 can contain A-Z, a-z, 0-9, +, /, and = for padding
+        # GitHub also uses - and _ in URL-safe base64
+        if not re.match(r'^[A-Za-z0-9+/\-_=]+$', suggestion_id):
```

Ruff (0.12.2)

Undefined name `re`

(F821)